mitodl/mit-open
# Better Headings Structure

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5242

### Description (What does it do?)
Adds better headings to our pages.

### Screenshots (if appropriate):

**Home** / **Search**

<img width="250" alt="Screenshot 2024-08-28 at 4 06 40 PM" src="https://github.com/user-attachments/assets/a591861a-f609-4565-a265-593777785fcd"> 
<img width="250" alt="Screenshot 2024-08-28 at 4 11 00 PM" src="https://github.com/user-attachments/assets/d9c7ea95-18f4-45ba-a332-c31d862f0b0d">

 **Unit / Department / Topic Channel**

<img width="250" alt="Screenshot 2024-08-28 at 4 06 28 PM" src="https://github.com/user-attachments/assets/defff034-e5c4-4282-b250-367f63595bcf">
<img width="250" alt="Screenshot 2024-08-28 at 4 11 28 PM" src="https://github.com/user-attachments/assets/2d5e7ae3-ed95-4cea-b82e-6e295ce2806e">
<img width="250" alt="Screenshot 2024-08-28 at 4 11 51 PM" src="https://github.com/user-attachments/assets/b0df385a-db11-44e0-807a-ba6d9a1b8fc8">


**Unit / Department / Topics / Listing**
<img width="250" alt="Screenshot 2024-08-28 at 4 12 13 PM" src="https://github.com/user-attachments/assets/56a80d88-9cce-4e71-a053-db4e608034ca"><img width="250" alt="Screenshot 2024-08-28 at 4 06 13 PM" src="https://github.com/user-attachments/assets/73532661-a414-4ffd-825b-78d46c50db9f"><img width="250" alt="Screenshot 2024-08-28 at 4 12 35 PM" src="https://github.com/user-attachments/assets/c4b2aa2e-42ff-4488-b9e4-fd32fc4d240b">

**About Us** / **Privacy**

<img width="250" alt="Screenshot 2024-08-28 at 4 13 41 PM" src="https://github.com/user-attachments/assets/7c2c07b4-58fb-46cd-8c2c-3d15e7d75f69">
<img width="250" alt="Screenshot 2024-08-28 at 4 13 57 PM" src="https://github.com/user-attachments/assets/0c942a44-4174-4628-ae26-20dddf9d6e5d">


### How can this be tested?
Check that:
1. There are no visible changes to the site. *There shouldn't be... if you notice any, it is unintential*.
2. The headings on each page make sense. Some ways you could view the headings:
    - HeadingsMap extension... [Chrome](https://chromewebstore.google.com/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi) ... I think it exists for other browsers, too
    - Or use a screenreader like MacOS Voiceover. With Voiceover, `VO + U` (where VO = voiceover key combo) opens the "Rotor". Then arrow keys nav to the headings navigation section.
